kernelci/build.py

```diff
@@ -229,7 +229,7 @@ def git_describe(tree_name, path):
     The returned value is a string with the "git describe" for the commit
     currently checked out in the local git repository.
     """
-    describe_args = "--match=v\*" if tree_name == "arm-soc" else ""
+    describe_args = "--match=v\*" if tree_name == "soc" else ""
```

Comment:
By the way, is this special treatment still required to get a useful git describe?

Reply:
Probably not, but I didn't want to remove that as part of this patch.  That should be investigated and removed separately.

Reply:
OK